website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +62,28 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
 
 Icons within `neutral` and `highlight` Toasts can be replaced with other icons. Change them only when the new icon provides the user with extra value; otherwise, use the default icon provided.
 
-<Doc::Layout @spacing="12px" @direction="vertical">
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px will take up too much of the screen space.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts should appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. Because of the placement of Toasts, any page-specific feedback to the user should be directly associated with the UI element. 
 
 ![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
 
 ### Displaying multiple Toasts
 
-When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast.
+When displaying multiple Toasts, they should stack vertically with a 16px margin between each Toast. For consistency, each stacked Toast should be the same width.
+
+![Multiple toast examples stacked](/assets/components/toast/toast-placement-multiple.png =600x*)
+
+## When to persist or time-out
+
+A Toast can either persist (until dismissed) or automatically dismiss (time-out) after seven seconds. If a Toast contains actions or critical information (often associated with warning or critical toasts, but not limited to), it is recommended to persist until the user dismisses it, otherwise, it should time-out after 7 seconds.
```

Comment:
Where did "7 seconds" come from? How was this number determined? Can that be communicated in the documentation so consumers understand it isn't an arbitrary number?

Reply:
Long version:[Reading comprehension analysis](https://en.wikipedia.org/wiki/Words_per_minute#Reading_and_comprehension) from this wiki that also [cites this book](https://iovs.arvojournals.org/article.aspx?articleid=2166061). Shorter answer, this number is around the industry standard where I see 6/7 seconds in other design systems. Especially with toasts where it triggers where folks may not expect, it gives the user enough time to see it then read it.

I can elaborate more.